Trustroots/trustroots
# Sentry: add Mongo/Mongoose integration

See:

- https://docs.sentry.io/platforms/node/performance/database/
- https://github.com/getsentry/sentry-javascript/pull/3252


**To test**
- Configure Sentry
- Cause MongoDB error
- Observe tracing from Sentry
